packages/compiler/core/checker.ts

```diff
@@ -334,13 +426,15 @@ export function createChecker(program: Program): Checker {
       return templateInstantiation[index];
     }
 
-    return createType({
+    return createAndFinishType({
```

Comment:
createType has been split into createType, finishType, and createAndFinishType which does both. With the new type prototypes which provide the `projections` member, we needed a way to initialize a type with the proper structure without actually invoking any decorators on it.